src/main.py

```diff
@@ -524,31 +532,22 @@ def loop():
 				player.play_speech(resources_path + 'alexayes.mp3')
 				time.sleep(0.5)
```

Comment:
Nothing anymore. I temporarily made the playback call non-blocking and had to wait here and then forgot about this. Great catch! I believe we're now back at the trigger-to-listen speed as before. It's been really slow as you've pointed out somewhere. And now we're back baby!